examples/svg/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+extern crate wasm_bindgen;
+
+use wasm_bindgen::prelude::*;
+
+// Definitions of the functionality available in JS, which wasm-bindgen will
+// generate shims for today (and eventually these should be near-0 cost!)
+//
+// These definitions need to be hand-written today but the current vision is
+// that we'll use WebIDL to generate this `extern` block into a crate which you
+// can link and import. There's a tracking issue for this at
+// https://github.com/rustwasm/wasm-bindgen/issues/42
+//
+// In the meantime these are written out by hand and correspond to the names and
+// signatures documented on MDN, for example
+#[wasm_bindgen]
+extern "C" {
+    type HTMLDocument;
+    static document: HTMLDocument;
+    #[wasm_bindgen(method)]
+    fn createElementNS(this: &HTMLDocument, namespaceURI: &str, qualifiedName : &str) -> Element;
+
+    #[wasm_bindgen(method, getter)]
+    fn body(this: &HTMLDocument) -> Element;
+
+    type Element;
+    #[wasm_bindgen(method, js_name = setAttribute)]
+    fn set_attribute(this: &Element, name: &str, value: &JsValue );
+    #[wasm_bindgen(method, js_name = appendChild)]
+    fn append_child(this: &Element, other: &Element);
+}
+
+// Called by our JS entry point to run the example
+#[wasm_bindgen]
+pub fn run() {
+    let svg = document.createElementNS("http://www.w3.org/2000/svg", "svg"); 
+    svg.set_attribute("width", &JsValue::from(200));
+    svg.set_attribute("height", &JsValue::from(100));
+    document.body().append_child(&svg);
+
+    let path = document.createElementNS("http://www.w3.org/2000/svg", "path");
+    path.set_attribute("d", &JsValue::from("M100,0 L200,100 100,200 0,100Z"));
+    path.set_attribute("fill", &JsValue::from("red"));    
+    svg.append_child(&path);    
```

Comment:
I think we want something that is slightly more compelling than a red diamond (I think that's what this is?)

How about porting

```svg
  <svg width="100%" height="100%">
    <rect width="100%" height="100%" fill="red" />
    <circle cx="100%" cy="100%" r="150" fill="blue" stroke="black" />
    <polygon points="120,0 240,225 0,225" fill="green"/>
    <text x="50" y="100" font-family="Verdana" font-size="55"
          fill="white" stroke="black" stroke-width="2">
            Hello!
    </text>
  </svg>
```

from https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Adding_vector_graphics_to_the_Web to `web-sys`?